.github/workflows/js-api-build.yml

```diff
@@ -80,8 +90,30 @@ jobs:
             #   shell: bash
             #   run: ./scripts/test-integration.sh || exit 1
 
+    test-performance:
+        needs: build
+        name: test-performance
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+              with:
+                fetch-depth: 2
+
+            - name: node
+              uses: actions/setup-node@v3
+              with:
+                node: 16
```

Comment:
Can we use `.nvmrc` here as well? Or do we need to use `16` instead of `18` version for this action?

Reply:
that project is very old, and does not work with latest node, investigating why